### ISSUE TITLE
googlechrome:120.0.6099.130 This version download failed

### ISSUE THREAD

**Issue by dong-lufei** [ScoopInstaller/Extras#12509]
```
# dong @ lufei in E:\test [09:00:59] C:1
$ scoop update *
googlechrome: 120.0.6099.71 -> 120.0.6099.130
idea-ultimate: 2023.3.1-233.11799.300 -> 2023.3.2-233.13135.103
Updating 2 outdated apps:
Updating 'googlechrome' (120.0.6099.71 -> 120.0.6099.130)
Downloading new version
Starting download with aria2 ...
Download: 12/22 09:03:04 [ERROR] CUID#8 - Download aborted. URI=https://dl.google.com/release2/chrome/ackbiqhb2dq3vcz5fzDownload: Exception: [AbstractCommand.cc:312] errorCode=1 Network problem has occurred. cause:A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 343531|ERR |       0B/s|D:/scoop_user_apps/cache/googlechrome#120.0.6099.130#https_dl.google.com_release2_chrome_ackbiqhb2dq3vcz5fzv7bivvpj5q_120.0.6099.130_120.0.6099.130_chrome_installer.exe_dl.7z
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 1) An unknown error occurred
ERROR https://dl.google.com/release2/chrome/ackbiqhb2dq3vcz5fzv7bivvpj5q_120.0.6099.130/120.0.6099.130_chrome_installer.exe#/dl.7z
    referer=https://dl.google.com/release2/chrome/ackbiqhb2dq3vcz5fzv7bivvpj5q_120.0.6099.130/120.0.6099.130_chrome_installer.exe#/
    dir=D:\scoop_user_apps\cache
    out=googlechrome#120.0.6099.130#https_dl.google.com_release2_chrome_ackbiqhb2dq3vcz5fzv7bivvpj5q_120.0.6099.130_120.0.6099.130_chrome_installer.exe_dl.7z

ERROR & 'D:\scoop_user_apps\apps\aria2\current\aria2c.exe' --input-file='D:\scoop_user_apps\cache\googlechrome.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.4 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=4 --split=32 --max-connection-per-server=16 --min-split-size=1M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=10372 --continue --summary-interval=0 --auto-save-interval=1

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=googlechrome%40120.0.6099.130%3a+download+via+aria2+failed
# dong @ lufei in E:\test [09:03:04] C:1
```

**Comment by HUMORCE** [comment]
Cannot reproduce.

```
> scoop install googlechrome
Installing 'googlechrome' (120.0.6099.130) [64bit] from extras bucket
120.0.6099.130_chrome_installer.exe (106.3 MB) [==============================================================] 100%
Checking hash of 120.0.6099.130_chrome_installer.exe ... ok.
Extracting dl.7z ... done.
Running installer script...
Linking ~\scoop\apps\googlechrome\current => ~\scoop\apps\googlechrome\120.0.6099.130
Creating shim for 'chrome'.
Creating shortcut for Google Chrome (chrome.exe)
Persisting User Data
Running post_install script...
'googlechrome' (120.0.6099.130) was installed successfully!
```

---

This may caused with rule-based proxies, check your proxy server if you set up a proxy for Scoop.

`dl.google.com` was not blocked by the GFW, in my memoery.

**Comment by HUMORCE** [comment]
Temporarily closed, feel free to leave a comment below.

**Reasons:**
- Cannot reproduce.
- No response and additional information
